src/topology/continuous_function/compact.lean

```diff
@@ -268,6 +278,34 @@ classical.some_spec (classical.some_spec (uniform_continuity f ε h)) w
 
 end uniform_continuity
 
+end continuous_map
+
+section comp_left
+variables (X : Type*) {𝕜 β γ : Type*} [topological_space X] [compact_space X]
+  [nondiscrete_normed_field 𝕜]
+variables [normed_group β] [normed_space 𝕜 β] [normed_group γ] [normed_space 𝕜 γ]
+
+open continuous_map
+
+/--
+Postcomposition of continuous functions into a normed module by a continuous linear map is a
+continuous linear map.
+Transferred version of `continuous_linear_map.comp_left_continuous_bounded`,
+upgraded version of `continuous_linear_map.comp_left_continuous`,
+similar to `linear_map.comp_left`. -/
+protected def continuous_linear_map.comp_left_continuous_compact (g : β →L[𝕜] γ) :
```

Comment:
can you register a simp lemma on how it acts on functions?

Reply:
At the moment I have a simp-lemma reducing `g.comp_left_continuous_compact X` to `g.comp_left_continuous 𝕜 X`, and the latter has simp-lemmas generated by `@[simps]`.  Is more needed?  (I never know whether to fill in the "extra" simp-lemmas that seem like they could be generated by combinations of existing ones.)

Reply:
As far as I can tell, there are not enough lemmas to make this work:
```
example (g : β →L[𝕜] γ) (f : C(X, β)) (x : X) :
  g.comp_left_continuous_compact X f x = g (f x) :=
by simp
```
fails. The reason is that you have a simp lemma saying how the coercion to `→ₗ[𝕜]` behaves, but this coercion does not show up in the example above, so there is no reason why the simp lemma should fire up.

Reply:
Thanks for the explanation!  I added
```
g.comp_left_continuous_compact X f x = g (f x)
```
as a simp-lemma.  Presumably,
```
⇑(g.comp_left_continuous_compact X) = ⇑(g.comp_left_continuous 𝕜 X)
```
would have worked instead, if I had wanted to pursue my previous "transitivity" theory.